lib/formats/carto/api_key/grants.json

```
@@ -73,6 +73,9 @@
             "name": {
               "type": "string"
             },
+            "owner": {
```

Should we raise an error if we get the `owner` in the POST or PUT methods of the `api_keys` API?

Since it's a public API someone might think they can set the owner for existing tables and try to force it via the API, but this attribute is actually just `read-only`, so we could just raise an error in that case.

---

AFAIK you can't update an API Key so we should no add that by now

---

Sorry I meant on creation

---

Ok, I'll add an error on that case

---

From the RFC:
```If "readOnly" has a value of boolean true, it indicates that the value of the instance is managed exclusively by the owning authority, and attempts by an application to modify the value of this property are expected to be ignored or rejected by that owning authority. ```

So it could be ignored, which is the behavior used by our `json-schema` library, so we are going to ignore it as well